tensorboard/webapp/settings/_redux/settings_reducers.ts

```diff
@@ -31,7 +31,7 @@ function settingsReady(state: SettingsState): boolean {
 }
 
 // Auto reload period cannot be lower than 30s to prevent server load.
-const MIN_RELOAD_PERIOD_IN_MS = 30000;
+export const MIN_RELOAD_PERIOD_IN_MS = 30000;
```

Comment:
The 30s was the default min value in Polymer IIRC. Because these numbers was spread, at this point, I am not even sure what is supposed to be the minimum and, here, I chose to conform everything to 30s in an arbitrary manner.